src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -141,9 +141,11 @@
   </Target>
 
   <Target Name="FailIfLowerThanMSBuild14"
-          Condition=" $(SonarQubeTempPath) != '' AND $(MSBuildToolsVersion) != '14.0' AND $(MSBuildToolsVersion) != '15.0'"
+          Condition=" $(SonarQubeTempPath) != '' AND $(MSBuildToolsVersion) != '14.0' AND $(MSBuildToolsVersion) != '15.0' AND $(MSBuildToolsVersion) != 'Current'"
```

Comment:
We don't have any tests that check this - not easy to do, as you can't set _MSBuildToolsVersion_ explicitly in a project file as MSBuild complains that it's a reserved property. The only way to automate it would be to execute a build using an older version of MSBuild, which would require more work on the test framework.

I've manually tested it.